### DISCUSSION TITLE
FLUX Inpainting Not Working With Latest Update?

### DISCUSSION THREAD

**Discussion by mistermcluvin** [lllyasviel/stable-diffusion-webui-forge#1985] [Q&A] [answered]
I updated Forge this AM and I noticed that FLUX Inpainting no longer works correctly for me. I get this dark, burnt, over saturated output? I swear it was working fine 2 days ago but I didn't use it after yesterdays update so I am not sure when the change occurred.

**Comment by abzaloff** [comment, marked as answer]
Everything is working well with the latest update. Check the CFG or other parameters

**Comment by JairVianna** [comment]
I am having the same problem, what you do to the Flux Inpainting back to work?
